source/features/parse-backticks.tsx

```diff
@@ -11,21 +11,24 @@ function init(): void {
 		'.commit-desc', // `isCommit`, `isCommitList`, `isRepoTree`
 		'.commit-message', // Pushed commits in `isPRConversation`, `isCompare`, `isReleasesOrTags`
 		'.message', // `isCommitList`, `isRepoTree`, `isBlame`
-		'.Box--condensed .link-gray[href*="/commit/"]', // `isSingleFile`
+		'.repository-content .js-details-container .link-gray[href*="/commit/"]', // `isSingleFile`
+		'.repository-content .js-details-container pre', // `isSingleFile`
 		'[aria-label="Issues"][role="group"] .js-navigation-open', // `isConversationList`
 		'[id^=ref-issue-]', // Issue references in `isIssue`, `isPRConversation`
 		'[id^=ref-pullrequest-]', // PR references in `isIssue`, `isPRConversation`
 		'.TimelineItem-body > del, .TimelineItem-body > ins', // Title edits in `isIssue`, `isPRConversation`
-		'.js-pinned-issue-list-item > span', // Pinned Issues
+		'.js-pinned-issue-list-item > .d-block', // Pinned Issues
 		'.pulse-section li', // `isPulse`
-		'.issues_labeled .text-gray-dark > a', // `isDashboard` "help wanted" event titles
 		'.js-recent-activity-container .text-bold', // `isDashboard`"Recent activity" titles
 		'.commits blockquote', // Newsfeed commits
 		'.release-header', // `isReleasesOrTags` Headers
-		'.Box-row.js-navigation-item a.link-gray-dark', // `isGlobalConversationList`
```

Comment:
I don't know why #2947 added this, it seems to be included in `[aria-label="Issues"][role="group"] .js-navigation-open` above

Reply:
If it works, then it works!
I don't remember, that commit was BC (before corona) 😃